### ISSUE TITLE
PM2 god daemon dying without any indication

### ISSUE THREAD

**Issue by geekyme** [Unitech/PM2#1489]
This has happened a few times already, usually once every few days. When I run my app in cluster mode with pm2, after a few days I will see that my node processes are no longer running. When I run `pm2 logs`, I see something like: 

```
[PM2] Spawning PM2 daemon
[PM2] PM2 Successfully daemonized
```

It seems like the PM2 daemon somehow died and killed all the node processes started along with it too.

These are the last few lines of my pm2.log

```
2015-08-02 11:26:03: [PM2][WORKER] Started with refreshing interval: 30000
2015-08-02 11:26:03: [[[[ PM2/God daemon launched ]]]]
2015-08-02 11:26:03: BUS system [READY] on port /home/carousell/.pm2/pub.sock
2015-08-02 11:26:03: RPC interface [READY] on port /home/carousell/.pm2/rpc.sock
2015-08-02 11:26:03: Starting execution sequence in -cluster mode- for app name:index id:0
2015-08-02 11:26:03: App name:index id:0 online
2015-08-02 11:26:03: Starting execution sequence in -cluster mode- for app name:index id:1
2015-08-02 11:26:03: App name:index id:1 online
2015-08-02 11:26:04: Starting execution sequence in -cluster mode- for app name:index id:2
2015-08-02 11:26:04: App name:index id:2 online
2015-08-02 11:26:04: Starting execution sequence in -cluster mode- for app name:index id:3
2015-08-02 11:26:04: App name:index id:3 online
```

I couldn't see any tell tale signs why my PM2 daemon is gone. What am I missing?

**Comment by jshkurti** [comment]
These logs seems to be related to the time you started PM2 again. I would rather have a look at the logs at the time PM2 crashed instead if possible. Thanks

**Comment by jshkurti** [comment]
Which version of PM2 are you using ?

**Comment by geekyme** [comment]
Yes the problem is, that's the last few lines of the log. There is no indication of any logs of PM2 crashing anywhere. 

I am using 0.14.3

**Comment by jshkurti** [comment]
It has been fixed on v0.14.5. Please upgrade your PM2 :)
